pyproject.toml

```diff
@@ -256,8 +256,10 @@ ignore-decorators = [
 "tools/graphql_codegen/**" = ["N802"]
 
 # On generated code:
-"wandb/sdk/automations/_generated/**" = [
-    "N8", # Ignore naming rules, as names from the GraphQL schema may not respect PEP8
+"wandb/**/_generated/**" = [
+    "N8",    # Ignore naming rules, as names from the GraphQL schema may not respect PEP8
+    "UP006", # Avoid using standard library generics e.g. `list[...]`, as this can cause issues with earlier pydantic versions
+    "UP007", # Avoid using `X | Y` for union fields, as this can cause issues with pydantic < 2.6
```

Comment:
good point! note to self: we should add these linting rules for settings and metadata as well.